lib/6to5/to-fast-properties.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * A trick from Bluebird to force V8 to use fast properties for an object.
+ * Read more: http://stackoverflow.com/questions/24987896/
+ *
+ * Use %HasFastProperties(obj) and --allow-natives-syntax to check whether
+ * a particular object already has fast properties.
+ */
+module.exports = function toFastProperties(obj) {
+  /*jshint -W027*/
+  function f() {}
+  f.prototype = obj;
+  return f;
+  eval(obj);
```

Comment:
Ya, that must be it. The eval, makes this library a non-option in some strict client side environments. a `try { } catch { }` likely accomplishes the same without the issue i describe.

@petkantonov care to share your wisdom?